src/Impostor.Api/Innersloth/GameOptions/RoleOptions/ScientistRoleOptions.cs

```diff
@@ -0,0 +1,33 @@
+namespace Impostor.Api.Innersloth.GameOptions.RoleOptions;
+
+public class ScientistRoleOptions : IRoleOptions
+{
+    public ScientistRoleOptions(byte version)
+    {
+        Version = version;
+    }
+
+    public byte Version { get; }
```

Comment:
This property does not exist in the base game, and it seems unused. Why?

Reply:
Base game instead postfix the class with `V07`, there is no V8 yet but seems like implementing it with a property will be way cleaner

Reply:
We can always add this field later down the road, we don't need to anticipate what Innersloth could possibly do next

Reply:
It's part of GameOptions which is already versioned

Reply:
If it's already part of something that has a version, do the underlying objects really need a version too?

Reply:
Yes? How else would you handle a new field in ScientistRoleOptions for example?

Reply:
Basically what @miniduikboot said, but it's w/e.